### ISSUE TITLE
Cannot edit a link in wysiwyg field

### ISSUE THREAD

**Issue by pvdbroek** [area17/twill#1694]
## Description

When I use a custom @formField for a wysiwyg form, I cannot alter a link after it has been made. I can click on a link, edit it, but then the old link url not altered. 

### Steps to reproduce

This is my definition:

```
@twillBlockTitle('Text')
@twillBlockIcon('text')

@formField('wysiwyg', [
'name' => 'html',
'label' => 'Content',
'toolbarOptions' => [ [ 'header' => [false, 1, 2, 3, 4] ], 'bold', 'italic', 'underline', 'strike', 'link', ['align' => ''], ['align' => 'center'], ['align' => 'right'], 'list-ordered', 'list-unordered', [ 'indent' => '-1'], [ 'indent' => '+1' ] ],
'placeholder' => '',
'translated' => true
])
```

### Expected result

The old link url should be altered into the new one

### Actual result

The old link url is still there

### Versions

Twill version: 2.8.7
Laravel version: 8.83
PHP version: 8

### Screenshots:

![afbeelding](https://user-images.githubusercontent.com/495242/173827408-8e986a85-cfc1-4d93-93be-f6f1d5db7f58.png)

![afbeelding](https://user-images.githubusercontent.com/495242/173827484-57afaed7-d2b6-43f8-93b4-bbb0cc465da1.png)

![afbeelding](https://user-images.githubusercontent.com/495242/173827540-f1df1957-35ae-4956-a096-989f232c06b5.png)

When I use the original text block from twill, it is is working, but I want to use another wysiwyg toolbar. Is this a bug in the code or am I doing something wrong?

**Comment by kallefrombosnia** [comment]
Can you please check the browser developer console to see if there are any errors/warnings?

**Comment by pvdbroek** [comment]
Thanks for your response, there is no error or warning in the console.

**Comment by haringsrob** [comment]
Hey @pvdbroek,

I am trying to reproduce the issue but it seems all correct on my end.

Can you try to run `php artisan twill:update` as your assets might be behind? Also try clearing all browser caches.

![Screen Recording 2022-06-20 at 09 12 05](https://user-images.githubusercontent.com/866743/174545247-b48a6924-629f-4d6a-a102-62984954a7ed.gif)

**Comment by pvdbroek** [comment]
I'm having one related issue. After I've ran `php artisan twill:update`, there seems to be a "Collapse / Expand all" bug:

When I open a page, the option that is shown is "Collapse all", however, all fields are already collapsed:

![afbeelding](https://user-images.githubusercontent.com/495242/176140928-2818b5a0-a8e3-43e2-8ee6-d558732e6a93.png)

After I've clicked on the "Collapse all", the right option occurs:

![afbeelding](https://user-images.githubusercontent.com/495242/176141551-6614f2a3-1851-4d82-89cb-38fb4bd1059d.png)

When I click on the "Expand all", everything works fine.

There is no error or warning in the dev console. Twill version is 2.8.8.

Does this bug occur with other people as well or is it just me? Any idea's how to fix it?
